internal/clientconn/conn.go

```diff
@@ -250,3 +241,13 @@ func (c *conn) run(ctx context.Context) (err error) {
 		}
 	}
 }
+
+// Describe implements prometheus.Collector.
+func (l *conn) Describe(ch chan<- *prometheus.Desc) {
```

Comment:
Those methods are not called anywhere

Reply:
it should be solved right now - call from the listener